### ISSUE TITLE
Ideation: Your ideas for web + messaging apps go here!

### ISSUE THREAD

**Issue by autonome** [autonome/web-and-messaging#1]
Ideation on possible types of integrations in either direction.

* Bots?
* Browser extensions?
* Deep-linking?
* Games?
* Media?

How can usage in one complement the other, in either direction?

Feel free to leave a comment here on this issue, or leave more detailed information on the etherpad:

https://public.etherpad-mozilla.org/p/mozsprint-web-and-messaging

**Comment by sandunisuru** [comment]
We can easily integrate Facebook Messenger Bots using the Bot Framework.

**Comment by TharinduDilshan** [comment]
Yes. I think it is possible to create a browser extension for one platform across messaging apps.
Also, there is an extensions called All-in-One Messenger for google chrome to integrate the apps.

**Comment by autonome** [comment]
Interesting, thanks for adding that! The website is https://allinone.im/. Unfortunately it's only a Chrome app and is not open source. Hopefully they'll support Firefox in the future.

Looks like it is a messenger app primarily, and doesn't integrate with the browser. Maybe if they open the source, we could use it as the basis of further experimentation.

**Comment by arthurfilipe** [comment]
Hi there, I liked your project very well! I wonder if would be cool (and if would be possible, of course!) create an option to share Facebook posts in WhatsApp and/or Telegram, and vice versa too.

**Comment by sandunisuru** [comment]
@arthurfilipe  In Facebook App, There is an option called share external, isn't it?
